README.md

```diff
@@ -13,17 +18,14 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
 
 1. [Create an IBMid](https://cloud.ibm.com/registration).
 
-1. Email `contsto2@in.ibm.com`.
-   1. Include the subject line: `IBM Cloud Satellite Storage <storage-solution-name> Integration`*. For example: `IBM Cloud Satellite Storage LocalVolume Integration`. 
+1. Create an issue under [IBM Satellite Storage](https://github.com/IBM/ibm-satellite-storage/issues) repo.
+   1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`. 
    1. Provide a high level description of the storage solution with a link to the user documentation or public repository.
 
-1. [Complete the onboarding process](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding)
-
-1. [Complete the Certification Requirements checklist.](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding/co-sell-1/tech-cert/architectural-questions)
-
-
-## Creating and testing your deployment
+## Registering with IBM Cloud
```

Comment:
Why same heading at 2 places? line 17 & 25?

Reply:
Fixed the heading